### ISSUE TITLE
Unable to explore cloudwatch metrics

### ISSUE THREAD

**Issue by pderaaij** [grafana/grafana#20977]
**What happened**:
After installing Grafana 6.5.1 I am unable to explore metrics via the cloudwatch plugin.

**What you expected to happen**:
Explore metrics.

**How to reproduce it (as minimally and precisely as possible)**:
Load the explore screen and select a Cloudwatch datasource. It also occurs on the playground:
https://play.grafana.org/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22CloudWatch%22,%7B%7D,%7B%22mode%22:%22Metrics%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D

**Anything else we need to know?**:
Message in console:
```
Plugin component error 
Object { message: "Failed to find exported plugin component for query-ctrl-cloudwatch" }
```

**Environment**:
- Grafana 6.5.1
- Cloudwatch plugin
- User OS & Browser: Tried in firefox and Brave

**Comment by marefr** [comment]
Duplicate of #20805

**Comment by marefr** [comment]
Fix will be included in Grafana 6.5.2
